requirements_test.txt

```diff
@@ -3,4 +3,5 @@ mock
 freezegun
 pytest
 coverage
-BeautifulSoup4
\ No newline at end of file
+BeautifulSoup4
+autopep8
```

Comment:
`autopep8` non viene utilizzato nella run dei test, quindi potremmo escluderlo dal relativo file di requirements.

Reply:
Suggerirei di utilizzarlo poiché la gran parte dei fix li ho fatti con autopep8. 
Se avete delle alternative per me vanno benissimo, basta che le build finiscano "verdi" in modo facile ;)